controllers/binder.go

```diff
@@ -399,11 +411,12 @@ func (b *binder) updateContainer(container interface{}) (map[string]interface{},
 	if err != nil {
 		return nil, err
 	}
-	if !b.sbr.Spec.BindAsFiles {
+
+	if b.sbr.Spec.BindAsFiles == nil || !*b.sbr.Spec.BindAsFiles {
```

Comment:
please extract this boolean expression into a dedicated var, for improved readability

Reply:
Done here c80db92